src/controllers/v1.controller.ts

```diff
@@ -29,6 +29,7 @@ const DEFAULT_STICKINESS_PARAMS = {
   duration: 30, // seconds
   useRPCID: true,
   relaysLimit: 0,
+  whitelistOrigins: [],
```

Comment:
Variable name should be changed to reflect stickiness. Right now this sounds like the whitelistOrigins used for security.

stickyOrigins ?

Reply:
uhhmmn is already inside the `stickinessOptions` object, I think that would be redundant